NavigationReactNative/src/android/src/main/java/com/navigation/reactnative/ToolbarView.java

```diff
@@ -202,6 +205,19 @@ void setOnSearchListener(OnSearchListener onSearchListener) {
 
     }
 
+    void setCollapseSearchButton(ImageButton collapseSearchButton) {
+        this.collapseSearchButton = collapseSearchButton;
+        setCollapseSearchTintColor(tintColor);
+    }
+
+
+    void setCollapseSearchTintColor(Integer tintColor) {
+        if (tintColor != null)
+            collapseSearchButton.setColorFilter(tintColor);
+        else
+            collapseSearchButton.clearColorFilter();
+    }
```

Comment:
Nearly there!
You should remove the `tintColor` parameter because the `tintColor` is a property of the class 
Also, you're missing the check that `collapseSearchButton` is not null

Reply:
done!